### ISSUE TITLE
[darwin:yosemite] Scrolling causes current page to become blank

### ISSUE THREAD

**Issue by elliot-wood** [spolu/breach_core#116]
If I load a webpage and then proceed to scroll, the page immediately becomes blank:
![screen shot 2014-07-13 at 4 06 23 am](https://cloud.githubusercontent.com/assets/6581437/3562396/4a34785c-09ef-11e4-955b-7d915ec020f1.png)

However, if the page's content fits in the browser window entirely, I can scroll and get the OS X "rubber band" effect without the page blanking.

**Comment by zhiayang** [comment]
Can confirm this issue exists in 10.10.

**Comment by rainder** [comment]
+1

**Comment by Jonymul** [comment]
+1. Same issue here.

**Comment by drikin** [comment]
+1 same on my Yosemite DP3

**Comment by majorwaves** [comment]
+1

You can navigate from page to page fine, but as soon as you scroll it goes blank.

Yosemite DP3

**Comment by lorenzoferrante** [comment]
I also have this problem. It's unusable, please fix it, I really like it.

**Comment by Guitaraholic** [comment]
yep same here in OSX 10.10 ... is there any useful logs or additional information that could assist?

**Comment by octalmage** [comment]
I want to use this so bad. :(

**Comment by miketamis** [comment]
+1 makes breach unusable

**Comment by entepe85** [comment]
+1 here

**Comment by warsfade** [comment]
Same here, latest Yosemite beta.

**Comment by nicregis** [comment]
+1 on Yosemite Public Beta 4

**Comment by solocshaw** [comment]
+1

**Comment by tcgumus** [comment]
+1

**Comment by kunday** [comment]
Same here, unable to scroll in Yosemite.

**Comment by spolu** [comment]
Hi guys, it has been fixed in https://github.com/breach/thrust
The next version of Breach will definitely work on Yosemite.

**Comment by tcgumus** [comment]
We are waiting :) when the next update will come out?

**Comment by spolu** [comment]
Not clear at the moment, and it might look quite different actually. But feel free to join on #breach on freenode to chat and contribute! :+1:

**Comment by d0p3fish** [comment]
+1

unusable :(

**Comment by morganrallen** [comment]
And as previous comments have stated, this version will be superseded by
the working being done on Thrust.

On Tue, Dec 23, 2014 at 1:56 AM, d0p3fish notifications@github.com wrote:

> +1
> 
> unusable :(
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/breach/breach_core/issues/116#issuecomment-67936066.

## 

http://ithoughtyouweretherobot.com
Metal and Wire

http://nolonelyguineapigs.com/
Wandering and Rambling

http://morglog.org
Old and Neglected.

-----BEGIN PGP PUBLIC KEY BLOCK-----
Version: OpenPGP.js v.1.20121007
Comment: http://openpgpjs.org

xsBNBFG3btUBB/9+/WJNOSIuc/praKPaPOweqXV5s7PGRD+HAnNWF/19YAY3
AFtfeoelhhY4sMJoobTaJzcZojznZ1kl/7UuuYCnbJO3Z9kaSMVrVxEZMSLe
YmW8Hc4ZsTnl9f05DQFy8ABpAvMLmXJQXh63BbfzjFqNbSWQfLhrubSM2Elq
xmN6EsmRyAuEeYlSnal+Di2MlViohpCbbagE2D6AZTUECIamTib6+DuLG4XO
b4GGKHR9TXu2qS1VH6hqcvdEz2MDE5OqZzAnBgcF2dvcOdTk9gAOoR1T+qKj
IJtg2X0ETbxcPqEXQoXE8VrxN0MLT5JQKTg26+csU4P2w0rEWsUsJombABEB
AAHNJU1vcmdhbiBBbGxlbiA8bW9yZ2FucmFsbGVuQGdtYWlsLmNvbT7CwFwE
EAECABAFAlG3btwJEG9pFLKaVEfeAAD7hwf+K8LN/kfgi8GTRLiTcuM+hSbm
uAeiBY+VaJIQFLzoFlt8A0hBlLNPFNNmUDxdPrMErdHgZ7HtJW/6RANuBTti
5hr3EBguH1GQyK7BbUqJGvaEX4UNRej3uyX3ufXDsIB/3NsKaTyyu+SO0+Hv
Cn2DwmlfSXcPhoarx6Turizn/WwgtTN3tqrMWxxiCKWydLH5xYuz+23BLMdc
oF/zkbF+7ddFhmyfiFh4Ej9//tsfwPj2cACU3uLTrcwCo7IxUtkrqMF/fhlm
LEqjhzDDjq9v7GNWb/5xB2xgTaucJ7/ljuS8nPlZqwCI8NlTgSqC8nyVLyjS
yVtOzkItcS0UqyW1Mg==
=V/Ia
-----END PGP PUBLIC KEY BLOCK-----

**Comment by adamwong246** [comment]
+1 :disappointed:

**Comment by ethanve** [comment]
+1

I so want to use this!

**Comment by hacknightly** [comment]
+1

(OSX 10.10.1) REALLY looking forward to this.

**Comment by henryspivey** [comment]
+1 me too if I don't use a mouse. Using a mouse allows for scroll but touchpad causes page to become blank
